source-map.bs

```diff
@@ -4,7 +4,7 @@ H1: Source Map
 Shortname: source-map
 Level: 1
 Status: STAGE0
-URL: https://source-map.github.io/source-map-spec/
+URL: https://tc39.github.io/source-map-spec/
```

Comment:
I think I would go all the way and resolve the redirects to https://tc39.es/source-map/.

Reply:
Okay.